tests/stdlib/uselocks.nim

```diff
@@ -9,3 +9,7 @@ proc createMyType*[T]: MyType[T] =
 proc use* (m: var MyType): int =
   withLock m.lock:
     result = 3
+
+block:
+  var l: Lock
+  doAssert $l == "()"
```

Comment:
Why is this in `uselocks.nim` and not `tlocks.nim`?

Reply:
it should be mlocks:

https://nim-lang.github.io/Nim/contributing.html
> All test files are prefixed with t. If you want to create a file for import into another test only, use the prefix m.